src/utils/UserUtils.ts

```diff
@@ -4,20 +4,28 @@ export const getUserType = (user: User) => {
   if (user.type === 'pupil') {
     return `Schüler*in`;
   }
+  return `Student*in`;
+};
+
+export const getUserTags = (user: User) => {
+  const studentTags = {};
+  if (user.type === 'pupil') {
+    return { pupil: 'Schüler*in' };
+  }
 
   if (user.isTutor) {
-    return `Student*in`;
+    studentTags["isTutor"] = 'Tutor';
   }
 
   if (user.isInstructor) {
-    return 'Tutor*in';
+    studentTags["isInstructor"] =
+      user.university === null ? 'Kursleiter' : 'Praktikant';
```

Comment:
Here as well `Kursleiter*in` and `Praktikant*in`

Reply:
yes will change them